ocp4/profiles/coreos-ncp.profile

```diff
@@ -227,49 +220,51 @@ selections:
     #- package_abrt_removed
 
     ### Login
-    #- disable_users_coredumps
-    #- sysctl_kernel_core_pattern
-    #- coredump_disable_storage
-    #- coredump_disable_backtraces
-    #- service_systemd-coredump_disabled
+    - disable_users_coredumps
+    - sysctl_kernel_core_pattern
+    - coredump_disable_storage
+    - coredump_disable_backtraces
+    - service_systemd-coredump_disabled
     #- var_accounts_max_concurrent_login_sessions=10
     #- accounts_max_concurrent_login_sessions
     #- securetty_root_login_console_only
     #- var_password_pam_unix_remember=5
     #- accounts_password_pam_unix_remember
 
     ### SELinux Configuration
-    #- var_selinux_policy_name=targeted
-    #- selinux_policytype
+    - var_selinux_state=enforcing
+    - selinux_state
+    - var_selinux_policy_name=targeted
+    - selinux_policytype
 
     ### Application Whitelisting (RHEL 8)
-    #- package_fapolicyd_installed
-    #- service_fapolicyd_enabled
-    #- configure_fapolicyd_mounts
+    - package_fapolicyd_installed
+    - service_fapolicyd_enabled
+    - configure_fapolicyd_mounts
 
     ### Enable the Hardware RNG Entropy Gatherer Service
-    #- service_rngd_enabled
+    - service_rngd_enabled
 
     ### Configure SSSD
-    #- sssd_run_as_sssd_user
+    - sssd_run_as_sssd_user
 
     ### Configure USBGuard
-    #- service_usbguard_enabled
-    #- configure_usbguard_auditbackend
-    #- usbguard_allow_hub
-    #- usbguard_allow_hid
+    - service_usbguard_enabled
+    - configure_usbguard_auditbackend
+    - usbguard_allow_hub
+    - usbguard_allow_hid
 
     ### Enable / Configure FIPS
-    #- enable_fips_mode
-    #- var_system_crypto_policy=fips
-    #- configure_crypto_policy
-    #- harden_sshd_crypto_policy
-    #- harden_ssh_client_crypto_policy
-    #- configure_bind_crypto_policy
-    #- configure_openssl_crypto_policy
-    #- configure_libreswan_crypto_policy
-    #- configure_kerberos_crypto_policy
-    #- enable_dracut_fips_module
+    - enable_fips_mode
+    - var_system_crypto_policy=fips
+    - configure_crypto_policy
+    - harden_sshd_crypto_policy
+    - harden_ssh_client_crypto_policy
+    - configure_bind_crypto_policy
```

Comment:
I really hope nobody would ever run BIND on RHCOS directly :-)

Reply:
they don't. Also, Kubernetes comes with its own DNS service (CoreDNS), which runs on a container.

Reply:
@redhatrises if OpenShift comes with its own DNS service, would all the DNS SRG checks be applicable?

Reply:
@shawndwells @redhatrises are those checks applicable for DNS that apply to internal services? CoreDNS doesn't resolve external domains. For that, the deployment relies on whatever is configured for the private network. Which means for AWS, they might rely on Route53, or in a private cloud, whatever DNS nameserver is configured by the deployer.

Reply:
@shawndwells @JAORMX Let's not have this conversation here. Let's have this separately, but now we need to have that.